Support latest version of `rocket`

`rocket` updated to `0.5.0-rc.2`. It has breaking changes that leads to the broken build. We need to update the code so that the build starts to pass

Current errors:
```
error[E0432]: unresolved import `rocket::response::content::Custom`
Error:   --> bob/src/api/http.rs:16:16
   |
16 |     response::{content::Custom as Content, Responder, Result as RocketResult},
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^ no `Custom` in `response::content`

error[E0282]: type annotations needed
Error:    --> bob/src/api/http.rs:695:1
    |
695 | / async fn get_data(
696 | |     bob: &State<BobServer>,
697 | |     key: Result<DataKey, StatusExt>,
698 | | ) -> Result<Content<Vec<u8>>, StatusExt> {
    | |      -----^
    | |______|____|
    |        |    cannot infer type
    |        consider giving `___responder` a type
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -13,7 +13,7 @@ Bob versions changelog
 
 
 #### Updated
-- Update rocket to  (#486)
+- Update rocket to v0.5.0-rc.2 (#486)
 
 
 ## [2.1.0-alpha.2] - 2022-04-26

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -13,7 +13,7 @@ Bob versions changelog
 
 
 #### Updated
-
+- Update rocket to  (#486)
 
 
 ## [2.1.0-alpha.2] - 2022-04-26

--- a/bob-apps/Cargo.toml
+++ b/bob-apps/Cargo.toml
@@ -27,7 +27,7 @@ metrics = { version = "0.17", features = ["std"] }
 mockall = "0.10"
 prost = "0.9"
 ring = "0.16"
-rocket = "0.5.0-rc.1"
+rocket = "0.5.0-rc.2"
 serde = "1.0"
 serde_derive = "1.0"
 serde_yaml = "0.8"

--- a/bob/Cargo.toml
+++ b/bob/Cargo.toml
@@ -30,7 +30,7 @@ mockall = "0.10.2"
 pipers = "1.0.0"
 prost = "0.9"
 ring = "0.16"
-rocket = { version = "0.5.0-rc.1", features = ["json", "uuid"] }
+rocket = { version = "0.5.0-rc.2", features = ["json", "uuid"] }
 serde = "1.0"
 serde_derive = "1.0"
 serde_yaml = "0.8"

--- a/bob/src/api/http.rs
+++ b/bob/src/api/http.rs
@@ -13,7 +13,7 @@ use rocket::{
     data::ByteUnit,
     http::{ContentType, Status},
     request::FromParam,
-    response::{content::Custom as Content, Responder, Result as RocketResult},
+    response::{Responder, Result as RocketResult},
     serde::{json::Json, uuid::Uuid},
     Config, Data, Request, Response, Rocket, State,
 };
@@ -695,11 +695,12 @@ async fn read_directory_children(mut read_dir: ReadDir, name: &str, path: &str) 
 async fn get_data(
     bob: &State<BobServer>,
     key: Result<DataKey, StatusExt>,
-) -> Result<Content<Vec<u8>>, StatusExt> {
+) -> Result<(ContentType, Vec<u8>), StatusExt> {
     let key = key?.0;
     let opts = BobOptions::new_get(None);
     let result = bob.grinder().get(key, &opts).await?;
-    Ok(Content(infer_data_type(&result), result.inner().to_owned()))
+    println!("{:?}", result.inner());
+    Ok((infer_data_type(&result), result.inner().to_owned()))
 }
 
 #[post("/data/<key>", data = "<data>")]

--- a/bob/src/api/http.rs
+++ b/bob/src/api/http.rs
@@ -699,7 +699,6 @@ async fn get_data(
     let key = key?.0;
     let opts = BobOptions::new_get(None);
     let result = bob.grinder().get(key, &opts).await?;
-    println!("{:?}", result.inner());
     Ok((infer_data_type(&result), result.inner().to_owned()))
 }
 

